### ISSUE TITLE
Write hyperparameters to json file 

### ISSUE THREAD

**Issue by samdong2101** [uf3/uf3#104]
It will be helpful to write the hyperparameters used to train your model into your json files, so you always have a written account of what you did to train your model

**Comment by sunghjung3** [comment]
In case this is useful, it looks like @sxie22 wrote [this python file](https://github.com/uf3/uf3/blob/master/uf3/util/user_config.py) 2 years ago that reads hyperparameters from a `.yaml` file (like [this one](https://github.com/uf3/uf3/blob/master/uf3/default_options.yaml)) for training.

**Comment by monk-04** [comment]
I agree with @samdong2101 it will be helpful to save hyperparameters along with coefficients in the json file.

@samdong2101 Are you interested in developing the code?

**Comment by michaelmacisaac** [comment]
I agree with this, I do this for my models as well as output my model validation results, such as energy and force errors, elastic constants, energy per atom, etc.
